web_widget_table/static/js/web_widget_table.js

```diff
@@ -0,0 +1,55 @@
+/* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_widget_table', function (require) {
+    "use strict";
+
+    var registry = require('web.field_registry');
+    var AbstractField = require('web.AbstractField');
+    var FieldDataTable = AbstractField.extend({
+        className: 'oe_form_field_table',
+        supportedFieldTypes: ['char'],
+        init: function init () {
+            this._super.apply(this, arguments);
+        },
+        _render: function _render () {
+            this.$el.empty();
+            var field = this.nodeOptions.field_name,
+                fieldModel = this.record.data[field].model,
+                fieldRecordIds = this.record.data[field].res_ids,
+                table = "<table><thead><tr>",
+                table_fields = this.nodeOptions.fields,
+                headers = this.nodeOptions.headers,
+                datatable_params = this.nodeOptions.datatable_params;
+            headers.forEach(function (element) {
+                table += '<th>' + element + '</th>';
+            });
+            table += "</tr></thead><tbody>";
+
+            this._rpc({
```

Comment:
as far as I understand, you already have the field in the view, no?
If true, can't you simply read data from current record instead of issuing another request?